packages/amplify-category-auth/resources/overrides-resource/auth/override.ts.sample

```diff
@@ -1,5 +1,5 @@
-import { AmplifyAuthCognitoStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+import { AmplifyAuthCognitoStackTemplate, AmplifyProjectInfo } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyAuthCognitoStackTemplate, projectInfo: AmplifyProjectInfo) {
```

Comment:
I think the variable name should be called `amplifyProjectInfo` even with the type specifying it's Amplify since this var name gets used in the customer code.